src/decorator/live-decorator.ts

```diff
@@ -27,7 +27,7 @@ export class KalturaLiveEngineDecorator implements KalturaPlayerTypes.IEngineDec
     }
 
     get active(): boolean {
-        return this._plugin.isLiveEntry();
+        return this._plugin.player.isLive();
```

Comment:
are there another uses of `_plugin.isLiveEntry` ? is so need to fix them as well and remove `_plugin.isLiveEntry` api and flag.
But first let's try to understand why the plugin didnt use the player api from the beginning. maybe you will find something in history

Reply:
we use them in `src/middleware/live-middleware.ts`
but it seems like flag should be renamed to `isActive` that define if plugin already activated and added all the listeners.

Reply:
PR

Reply:
so in this issue the decorator will be active but the plugin not. does it make sense?